pkg/client/database/postgres/executor.go

```diff
@@ -51,6 +56,18 @@ func (e Executor) Finalize(_ context.Context, err error) error {
 	return err
 }
 
+// Validates that we can actually connect to the postgres database.
```

Comment:
```suggestion
// ValidatePostgresConnection validates that we can actually connect to the postgres database.
```
(valid godoc format as described [here](https://go.dev/blog/godoc))